WcaOnRails/app/jobs/sync_mailing_lists_job.rb

```diff
@@ -75,17 +34,9 @@ def perform
     GsuiteMailingLists.sync_group("sports@worldcubeassociation.org", Team.wsot.current_members.includes(:user).map(&:user).map(&:email))
     GsuiteMailingLists.sync_group("archive@worldcubeassociation.org", Team.wat.current_members.includes(:user).map(&:user).map(&:email))
 
-    SENIOR_DELEGATES_REGIONS_INFO.each do |region|
-      senior_delegates = User.senior_delegates.where("location like ?", region[:query])
-      if senior_delegates.length > 1
-        raise "Multiple Senior Delegates match #{region[:query]}"
-      elsif senior_delegates.empty?
-        raise "No Senior Delegate matches #{region[:query]}"
-      else
-        senior_delegate = senior_delegates.first
-      end
-      delegates = senior_delegate.subordinate_delegates
-      GsuiteMailingLists.sync_group(region[:mailing_list], (delegates + [senior_delegate]).map(&:email))
+    UserGroup.delegate_regions.each do |region|
+      emails = region.roles.map { |role| role[:user][:email] }
```

Comment:
Because `role` is currently not of `Role` type, instead it's just a JSON. It will be changed to `role.user.email` after the migration of delegates are completed.